modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -48,9 +48,7 @@ endif::ovn[]
 ====
 If your egress firewall includes a deny rule for `0.0.0.0/0`, access to your {product-title} API servers is blocked. You must include the IP address range that the API servers listen on in your egress firewall rules.
```

Comment:
Maybe you could just get rid of "You must include the IP address range that the API servers listen on in your egress firewall rules" and replace it with the sentence you added on line 51?